tests/stdlib/tos.nim

```diff
@@ -189,6 +189,14 @@ block walkDirRec:
 
   removeDir("walkdir_test")
 
+when not defined(windows):
```

Comment:
why do you exclude this test from windows? Symlinks should work on windows, too.

Reply:
Because `pcLinkToDir` is not supported on Windows afaict

Reply:
Symlinks on Windows require admin rights and are shit.